Landing bugs

Joshaw-k opened the pull request:
Closes #17

Commit:
mobile landing page fix

Commit:
mobile landing page fix

Commit:
mobile landing page fix

Commit:
mobile landing page fix

mercykolapo reviewed (commented):
The noise on the landing page disappeared.
Could you check again?
The header also changed. It is supposed to blur items passing underneath it, while sticking to the top.
It would be nice if you made a list of what you did also.

Commit:
fix header backdrop

Commit:
fix header backdrop

Commit:
fix header backdrop

Joshaw-k commented:
> The noise on the landing page disappeared. Could you check again? The header also changed. It is supposed to blur items passing underneath it, while sticking to the top. It would be nice if you made a list of what you did also.

Done.
1. fixed logo and hamburger spacing
2. aligned landing page texts and spacing to suit figma design
3. made navbar menu on mobile cover entire screen  when clicked and eliminated modals for launch dapp and company rather listing them out

faytey reviewed (approved):
Awesome, looks solid, thanks
